src/go/k8s/main.go

```diff
@@ -170,16 +170,14 @@ func main() {
 		os.Exit(1)
 	}
 
-	namespaceScoped := namespace != ""
-
 	configurator := resources.ConfiguratorSettings{
 		ConfiguratorBaseImage: configuratorBaseImage,
 		ConfiguratorTag:       configuratorTag,
 		ImagePullPolicy:       corev1.PullPolicy(configuratorImagePullPolicy),
 	}
 
 	// we are in v2 mode if we are namespace Scoped, otherwise we are in clusterScope or v1 mode
-	if namespaceScoped {
+	if namespace != "" { // nolint:nestif // this is ok
```

Comment:
This could be rebased (merged with previous commit) to have linear commit history

Reply:
will do, i will flatten things to avoid confusion